overops-functions/src/main/java/com/takipi/udf/microsoftteams/MicrosoftTeamsInput.java

```diff
@@ -0,0 +1,30 @@
+package com.takipi.udf.microsoftteams;
+
+import com.takipi.udf.input.Input;
+
+class MicrosoftTeamsInput extends Input {
+    public String url;
+
+    protected MicrosoftTeamsInput(String raw) {
+        super(raw);
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder builder = new StringBuilder();
+
+        builder.append("Microsoft Teams Alert ( url = ").append(url).append(" )");
+
+        return builder.toString();
+    }
+
+    public void checkUrl() {
+        if (url == null || !url.matches("^https.*outlook.*webhook.*")) {
```

Comment:
Please have each if condition in it's own parentheses.
i.e. in this case:
```
if ((url == null) || (!url.matches("^https.*outlook.*webhook.*"))) {
```

Reply:
done